web/convert/breakpoint.ts

```diff
@@ -0,0 +1,21 @@
+import type { MediaQuery } from 'typestyle/lib/types'
+import { UnistylesState } from '../state'
+import { keyInObject } from '../utils'
+import { isUnistylesMq, parseMq } from '../mq'
+import { Orientation } from '../../src/specs/types'
+
+export const convertBreakpoint = (breakpoint: string): MediaQuery => {
+    if (Object.values(Orientation).includes(breakpoint as Orientation)) {
```

Comment:
it's only available on mobile, you can remove it

Reply:
It's also available in web https://developer.mozilla.org/en-US/docs/Web/CSS/@media/orientation

Reply:
ah I see!